website/public/js/controllers/groupsCtrl.js

```diff
@@ -220,6 +220,13 @@ habitrpg.controller("GroupsCtrl", ['$scope', '$rootScope', 'Shared', 'Groups', '
       $scope.usernames = [];
     }
 
+    $scope.filterUser = function(item) {
```

Comment:
Why is the variable name item? Since items have a very specific meaning in this app, might be better to use another word.